### ISSUE TITLE
Can't log in with current calendar plugin

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#346]
```
What steps will reproduce the problem?
1. I'm only using "calendar" in the main.inc.php's plugin array.
If I log out, and try to log in, the main interface is struck (returns a blank 
page)
2. I'm using RC 0.8rc
3. If I change the main.inc.php not to include "calendar" in the list, the 
interface logs me in (it returns the interface instead of a blank page)
4. If I change back the plugin array in the main.inc.php file, once I'm logged 
in to include "calendar", the calendar plugin loads (but if I log out and try 
back in, it fails again).


I'm using the calendar plugin from the svn.


```

Original issue reported on code.google.com by `boite.po...@gmail.com` on 26 Jun 2012 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
Works for me. Anything in the error log?
```

Original comment by `rol...@liebl.ath.cx` on 26 Jun 2012 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
No, I've increased the log level, but apart from SQL queries going on, it stop 
logging.
Wireshark show it sends: 
HTTP/1.1 200 OK
Server: nginx/0.7.67
Date: Tue, 26 Jun 2012 15:05:49 GMT
Content-Type: text/html
Transfer-Encoding: chunked
Connection: keep-alive
X-Powered-By: PHP/5.3.14
Expires: Thu, 19 Nov 1981 08:52:00 GMT
Cache-Control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
Pragma: no-cache
Content-Encoding: gzip
Vary: Accept-Encoding

And that's it (no data)
```

Original comment by `boite.po...@gmail.com` on 26 Jun 2012 at 3:11

**Comment by GoogleCodeExporter** [comment]
```
I can't help you unless you grant me access to your system:

* IMAP test account
* Webmail URL
* FTP-/SSH-Access to Roundcube Root

Please mail details to myroundcube at mail4us dot net.
```

Original comment by `rol...@liebl.ath.cx` on 26 Jun 2012 at 3:16

**Comment by GoogleCodeExporter** [comment]
```
By enabling tracing, with xdebug, the script stops here:
   1,4280    6740720                     -> rcube_config->get() /srv/www/rc/program/include/rcmail.php:1296
    1,4281    6740752                       -> rcmail::get_instance() /srv/www/rc/program/include/rcube_config.php:189
    1,4282    6740784                       -> is_object() /srv/www/rc/program/include/rcube_config.php:194
    1,4283    6741088                       -> rcube_plugin_api->exec_hook() /srv/www/rc/program/include/rcube_config.php:196
    1,4284    6741120                         -> is_array() /srv/www/rc/program/include/rcube_plugin_api.php:258
    1.4625      14672
TRACE END   [2012-06-26 15:22:01]

So, I've put a var_dump in rcube_plugin_api.php:258 for all var in this method 
with var_dump($hook and $args) and I get this:
string(10) "config_get"
array(3) {
  'name' =>
  string(7) "plugins"
  'default' =>
  array(0) {
  }
  'result' =>
  array(1) {
    [0] =>
    string(8) "calendar"
  }
}
string(10) "config_get"
array(3) {
  'name' =>
  string(7) "plugins"
  'default' =>
  NULL
  'result' =>
  array(1) {
    [0] =>
    string(8) "calendar"
  }
}
string(10) "config_get"
array(3) {
  'name' =>
  string(4) "skin"
  'default' =>
  string(7) "default"
  'result' =>
  string(5) "larry"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(18) "jquery_ui_skin_map"
  'default' =>
  array(0) {
  }
  'result' =>
  array(0) {
  }
}
string(10) "config_get"
array(3) {
  'name' =>
  string(14) "jquery_ui_i18n"
  'default' =>
  array(1) {
    [0] =>
    string(10) "datepicker"
  }
  'result' =>
  array(1) {
    [0] =>
    string(10) "datepicker"
  }
}
string(10) "config_get"
array(3) {
  'name' =>
  string(11) "date_format"
  'default' =>
  string(5) "Y-m-d"
  'result' =>
  string(5) "Y-m-d"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(11) "cal_tempdir"
  'default' =>
  string(24) "./plugins/calendar/temp/"
  'result' =>
  string(5) "/tmp/"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(12) "upcoming_cal"
  'default' =>
  NULL
  'result' =>
  bool(false)
}
string(10) "config_get"
array(3) {
  'name' =>
  string(22) "calfilter_allcalendars"
  'default' =>
  string(21) "tous les événements"
  'result' =>
  string(21) "tous les événements"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(26) "event_filters_allcalendars"
  'default' =>
  array(0) {
  }
  'result' =>
  array(0) {
  }
}
string(10) "config_get"
array(3) {
  'name' =>
  string(15) "db_table_events"
  'default' =>
  string(6) "events"
  'result' =>
  string(6) "events"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(18) "db_sequence_events"
  'default' =>
  string(10) "events_ids"
  'result' =>
  string(10) "events_ids"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(7) "backend"
  'default' =>
  string(6) "caldav"
  'result' =>
  string(5) "dummy"
}
string(10) "config_get"
array(3) {
  'name' =>
  string(10) "devel_mode"
  'default' =>
  NULL
  'result' =>
  NULL
}

```

Original comment by `boite.po...@gmail.com` on 26 Jun 2012 at 3:33

**Comment by GoogleCodeExporter** [comment]
```
Credentials sent to both your address.
```

Original comment by `boite.po...@gmail.com` on 27 Jun 2012 at 9:47

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 9.6.1
```

Original comment by `rol...@liebl.ath.cx` on 27 Jun 2012 at 2:05
- Changed state: **Fixed**
